<?xml version="1.0" encoding="UTF-8"?>
<!-- Launch a topic switch. Written by Tiger Sachse. -->
<launch>
  <arg name="node_name" doc="The name of this node."/>
  <arg name="primary_topic" doc="The switch's primary topic."/>
  <arg name="secondary_topic" doc="The switch's secondary topic."/>
  <arg name="output_topic" doc="The switch's output topic."/>
  <arg name="topic_type" doc="The switch's topics' types."/>

  <node name="$(arg node_name)"
        pkg="ezrassor_topic_switch"
        type="topic_switch"
        output="screen">
    <param name="node_name" value="$(arg node_name)"/>
    <param name="primary_topic" value="$(arg primary_topic)"/>
    <param name="secondary_topic" value="$(arg secondary_topic)"/>
    <param name="output_topic" value="$(arg output_topic)"/>
    <param name="topic_type" value="$(arg topic_type)"/>
  </node>
</launch>
